Objects/stringlib/count.h

```diff
@@ -4,6 +4,11 @@
 #error must include "stringlib/fastsearch.h" before including this module
 #endif
 
+// gh-97982: Implementing asciilib_count() is not worth it, FASTSEARCH() does
+// not specialize the code for ASCII strings. Use ucs1lib_count() for ASCII and
+// UCS1 strings: it's the same than asciilib_count().
+#if STRINGLIB_MAX_CHAR > 0x7Fu
```

Comment:
Maybe `!STRINGLIB_IS_UNICODE || STRINGLIB_MAX_CHAR > 0x7Fu`?

`STRINGLIB_MAX_CHAR` was only used in `codecs.h` when `STRINGLIB_IS_UNICODE` is true.

Reply:
Oh thanks, that makes my PR shorter!